var/spack/repos/builtin/packages/py-scs/package.py

```diff
@@ -31,6 +31,12 @@ class PyScs(PythonPackage, CudaPackage):
     depends_on("py-numpy@1.7:", type=("build", "run"))
     depends_on("py-scipy@0.13.2:", type=("build", "run"))
 
+    # in newer pip versions --install-options does not exist
+    depends_on("py-pip@:23.0", when="+float32", type="build")
+    depends_on("py-pip@:23.0", when="+extra_verbose", type="build")
+    depends_on("py-pip@:23.0", when="+int32", type="build")
+    depends_on("py-pip@:23.0", when="+blas64", type="build")
```

Comment:
Honestly I'd be fine with always having this requirement just to make the package simpler.

Reply:
Done.